pandas/core/arrays/base.py

```diff
@@ -38,11 +38,21 @@ class ExtensionArray(object):
     * copy
     * _concat_same_type
 
-    Some additional methods are available to satisfy pandas' internal, private
-    block API:
+    An additional method and attribute is available to satisfy pandas'
+    internal, private block API.
 
-    * _can_hold_na
     * _formatting_values
+    * _can_hold_na
+
+    Some methods require casting the ExtensionArray to an ndarray of Python
```

Comment:
Merge snafu?

Reply:
No, I reordered `_formatting_values` and `_can_hold_na`, since one is a method and the other is an attribute.

Reply:
I meant this section specifically. This block is repeated starting on line 57. Or perhaps I'm missing something.

Reply:
This is really weird.  The copy on my machine is clean and fine, but the copy on GitHub has the repeat.  Maybe because I resolved conflicts with master using GitHub interface.  UGH.